containers/concent-api/clean-container.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+
+readarray temporary_dependencies < ${BASH_SOURCE%/*}/build-dependencies.txt
```

Comment:
`clean-container.sh` and `clean-pyenv-dependencies.sh` differ only with the name the the txt file. Please make the name parameter and remove one the scripts.